pytentiostat/tester.py

```diff
@@ -243,7 +253,9 @@ def experiment(config_data, board, a0, a2, d9):
             d9,

```

Comment:
Now pass them in main and in tester as *args instead of the individual objects! Learned something new.

I passed as args instead of kwargs because I just wanted to pass the objects here. They will be assigned values throughout the experiment in read/write.